test/check-storage

```diff
@@ -560,6 +560,10 @@ class TestStorage(MachineCase):
         m = self.machine
         b = self.browser
 
+        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1187481
```

Comment:
This bug needs to be cloned for RHEL 7. There should be a 'Clone' link in the bug above.